### PR TITLE
Add a general configuration with unit tests

### PR DESCRIPTION
**Pull request by voscob**
### Description

I've added a general configuration for the connector [MEX-250](https://yalantis.atlassian.net/browse/MEX-250) and a method to parse and validate this configuration.
I've added a unit test with the most common cases to check the validator [MEX-251](https://yalantis.atlassian.net/browse/MEX-251).
This configuration will be used for both connectors (source and destination).

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-oracle/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: config/general.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+package config
+
+import (
+	"github.com/conduitio/conduit-connector-oracle/config/validator"
+	"github.com/conduitio/conduit-connector-oracle/models"
+)
+
+// A General represents a general configuration needed to connect to Oracle database.
+type General struct {
```
Comment (maksenius) [inline config/general.go:23]: What do you think about using  connection string to connect to oracle?
For example
"<your username>/<your password>@service_name"

File: config/general.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+	// Password is the configuration of the password to connect to Oracle database.
+	Password string `json:"password" validate:"required,lte=30,oracle"`
+
+	// URL is the configuration of the path to the Oracle database.
+	URL string `json:"url" validate:"required"`
+
+	// Table is the configuration of the table.
+	Table string `json:"table" validate:"required,lte=128,oracle"`
+}
```
Comment (maksenius) [inline config/general.go:35]: Where we can schema?

File: config/general.go (hunk 1 above)
Comment (voscob) [reply config/general.go:23]: Did you mean to use a connection string to connect to a database like `localhost:1521/ORCLCDB`?

File: config/general.go (hunk 2 above)
Comment (voscob) [reply config/general.go:35]: ```In Oracle, users and schemas are essentially the same things. You can consider that a user is an account you use to connect to a database, and a schema is the set of objects (tables, views, etc.) that belong to that account.```
https://dba.stackexchange.com/a/37022

You can see a flow to create a new schema here:
https://docs.oracle.com/en/cloud/paas/exadata-express-cloud/csdbp/create-database-schemas.html#GUID-C20C60A5-A655-4CAA-957A-93BF28F4E5FA

So I don't think it's necessary to add an additional schema field.

File: config/general.go (hunk 1 above)
Comment (voscob) [reply config/general.go:23]: You're right.
The connection string will be like `username/password@address:port/dbname`.

**Pull request merged**